conf/config.neon

```diff
@@ -302,6 +302,16 @@ services:
 		tags:
 			- phpstan.parser.richParserNodeVisitor
 
+	-
+		class: PHPStan\Parser\VariadicMethodsVisitor
+		tags:
+			- phpstan.parser.richParserNodeVisitor
```

Comment:
This isn't sufficient just for the RichParser, it has to be done in `SimpleParser` too. Because it's relevant for sources that aren't analysed too.

Reply:
I am not sure I understand what you mean. 

I though doing https://github.com/phpstan/phpstan-src/pull/3534/files#diff-04a831854bbfa49602b74bbce1164b0f12d847200d418504d7dd11bf34702cc9R53-R54 is what you suggested initially.

do I need some neon config? I don't see any other config for a tag containing "simpleparser" or something

Reply:
You need to add the visitors in SimpleParser same way NameResolver is added there.

And yeah, it also needs to be in RichParser so the unit test should probably run for both.

Right now on 1.12.x variadic functions are properly detected in vendor code, but your PR breaks that.

Reply:
> You need to add the visitors in SimpleParser same way NameResolver is added there.

I think I already did. Added tests which prove it. Hopefully I got you right ;).